### PR TITLE
Integrating pytest framework for drift tests with ods-ci

### PR DESCRIPTION
**Pull request by nehachopra27**
This PR integrates the drift tests in this repo https://github.com/trustyai-explainability/trustyai-tests to run with ods-ci.
The logs are written in a log folder test-output/pytest-logs/drift_test.txt

### CODE REVIEW

File: ods_ci/tests/Tests/1200__ai_explainability/1203__trustyai_drift_metrics.robot [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+*** Test Cases ***
+Run Drift Metrics Tests
+    [Documentation]    Verifies that the Bias metrics are available in Metrics Console for a model which was
+    ...                 deployed prior to enabling the TrustyAI service
+    [Tags]    RHOAIENG-8163     Smoke
+    Run Drift Pytest Framework
+    ${result} =   Run Process  source ${VIRTUAL_ENV_NAME}/bin/activate && cd ${TRUSTYAI-TESTS_DIR} && poetry env use 3.9 &&  poetry install && poetry run pytest -v -s trustyai_tests/tests/${TESTS_FOLDER}/
+    ...    shell=true
+    ...    stderr=STDOUT
```
Comment (lugi0) [inline ods_ci/tests/Tests/1200__ai_explainability/1203__trustyai_drift_metrics.robot:29]: isn't this already run in `Run Drift Pytest Framework`?

**Comment by nehachopra27** [comment]
Verified here: https://opendatascience-jenkins/job/rhods/job/rhods-ci-pr-test/3230/console

File: ods_ci/tests/Tests/1200__ai_explainability/1203__trustyai_drift_metrics.robot [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    Log To Console     "Removing directory ${TRUSTYAI-TESTS_DIR}"
+    Remove Directory        ${TRUSTYAI-TESTS_DIR}    recursive=True
+
+Run Drift Pytest Framework
+    [Documentation]    Run Drift tests using poetry
+    Log To Console    Running Drift tests using pytest
+    ${result} =    Run Process  source ${VIRTUAL_ENV_NAME}/bin/activate && cd ${TRUSTYAI-TESTS_DIR} && poetry env use ${PYTHON_VERSION} && poetry install && poetry run pytest -v -s ./trustyai_tests/tests/${TESTS_FOLDER}/ && deactivate     # robocop: disable:line-too-long
+    ...    shell=true     stderr=STDOUT     stdout=${LOG_OUTPUT_FILE}
+    Log To Console    ${result.stdout}
+    IF    ${result.rc} != 0    FAIL    Tests failed due to ${result.stdout} failed
```
[the remark this answers is not in the document]
Comment (tonyxrmdavidson) [reply ods_ci/tests/Tests/1200__ai_explainability/1203__trustyai_drift_metrics.robot:68]: To get past these linting errors you could try something like this

[the remark this answers is not in the document]
Comment (tonyxrmdavidson) [reply ods_ci/tests/Tests/1200__ai_explainability/1203__trustyai_drift_metrics.robot:68]: IF     ${result.rc} != 0
        FAIL   Tests failed due to ${result.stdout}
END

[the remark this answers is not in the document]
Comment (tonyxrmdavidson) [reply ods_ci/tests/Tests/1200__ai_explainability/1203__trustyai_drift_metrics.robot:68]: I don't think it is a big deal though, just a suggestion.

**Review by tonyxrmdavidson — APPROVED** [review]
Apart from the linting suggestions this /lgtm for me.  
I don't think the linting is a big enough deal to stop it getting merged.

File: ods_ci/tests/Tests/1200__ai_explainability/1203__trustyai_drift_metrics.robot (hunk 1 above)
Comment (nehachopra27) [reply ods_ci/tests/Tests/1200__ai_explainability/1203__trustyai_drift_metrics.robot]: Removed now

**Review by aloganat — APPROVED** [review]
lgtm

**Pull request merged**